Makefile

```diff
@@ -3,6 +3,10 @@
 # 	all be in $DEIS/vendor
 SHORT_NAME=etcd
 
+ifndef BUILD_TAG
+  BUILD_TAG = git-$(shell git rev-parse --short HEAD)
+endif
```

Comment:
can't this all be replaced with `BUILD_TAG ?= git-$(shell git rev-parse --short HEAD)`